src/help/guides/getting-started.md

```diff
@@ -63,42 +64,42 @@ This requires iOS 13.3+ or iPadOS 13.4+, the app will CRASH otherwise. Paperback
    <el-tag type="warning">Not the Windows Store app!</el-tag>
 1. Download and install [iCloud for Windows 8](https://support.apple.com/en-us/HT204283).
    <el-tag type="warning">Not the Windows Store app!</el-tag>
-   > Can't find the link to download **iCloud** for Windows 8? [Image](https://imgur.com/a/P1ef4Wd)
-1. Restart your PC after installing both **iTunes** and **iCloud**.
+  > Can't find the link to download **iCloud** for Windows 8? [GIF](https://imgur.com/a/P1ef4Wd)
+1. Restart your PC after you've installed both **iTunes** and **iCloud**.
    <el-tag type="warning">You need to restart your computer!</el-tag>
 1. Extract the downloaded **`AltInstaller.zip`** file. Run **`setup.exe`** and follow the instructions.
-1. Launch **AltServer** from your *Notification Area*/*System Tray* or *Hidden Icons*.
+1. Launch **AltServer** from your *System tray* or *Hidden icons*.
```

Comment:
I'm not sure but it seems `*System tray*` was the old name for `*Notification Area*`
https://docs.microsoft.com/en-us/windows/win32/shell/notification-area
> The notification area has been known historically as the system tray or status area.

Reply:
I guess I'm old lol

Reply:
Fixed in d2a4a6fc39706f5154330cb130e3d8ad2f099314